crypto/x509v3/v3_ncons.c

```diff
@@ -9,6 +9,7 @@
 
 #include <stdio.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
Ah! I'm not at my desk right now, and am not in position to make adequately weighed assessments. Apologies for distraction.